src/senaite/sync/fetchstep.py

```diff
@@ -56,6 +56,7 @@ def verify(self):
         storage["credentials"]["url"] = self.url
         storage["credentials"]["username"] = self.username
         storage["credentials"]["password"] = self.password
+        storage["credentials"]["content_types"] = self.content_types
```

Comment:
Not sure why the selected `content_types` need to be in the `credentials` storage... Please choose a better location to increase clarity

Reply:
I had had to move it to the storage after I saw need of those 'content types' in both- fetch and import steps. It cannot be just a variable or saved in memory, because currently a user can do `fetch`, then stop the instance, and then run the `import`...
Actually currently @juangallostra is working on making `settings` and `registry` records optional, so we will have to save them somewhere as well. 
So any better idea where to save all these _Import Options_ ?

Reply:
I meant that you should better put it into the `settings` storage;) The `credentials` should hold only the credentials for the sync.
 https://github.com/senaite/senaite.sync/blob/master/src/senaite/sync/browser/views.py#L142